examples/lang/hetzner0.mcl

```diff
@@ -0,0 +1,12 @@
+# simple demo of the Hetzner resource
+hetzner "resourcetest" {
+	apitokenpath => "../mgmt-dev/apitoken.txt", # local file
```

Comment:
You may want to consider instead using one of the mcl file reading functions, and instead have this struct API take a string... This way you can pass in a raw string _or_ a file, and you aren't forced to read from a file on disk... You can instead read from the deploy. Try using:

```
import "deploy"
deploy.readfile(...)
```

or
```
import "os"
os.readfile(...)
```

Reply:
Thanks, I agree that this is a cleaner solution.